Makefile

```diff
@@ -21,6 +21,16 @@
 export APPDIR = $(CURDIR)
 include $(APPDIR)/Make.defs
 
+# The GNU make CURDIR will always be a POSIX-like path with forward slashes
+# as path segment separators.  This is fine for the above inclusions but
+# will cause problems later for the native build.  If we know that this is
+# a native build, then we need to fix up the APPDIR path for subsequent
+# use
+
+ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+APPDIR = ${shell echo %CD%}
```

Comment:
if so, should we replace line 21 directly

Reply:
do you means change Line 31: `APPDIR = ${shell echo %CD%}` to export `APPDIR = ${shell echo %CD%}` ?

Reply:
Yes, you are right.